packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesController.kt

```diff
@@ -191,6 +196,11 @@ class UpdatesController private constructor(
    */
   @Synchronized
   fun start(context: Context) {
+    if (isStarted) {
+      return
```

Comment:
One alternative is to throw an IllegalStateException here. With the other measures in place, we never should reach this code because the singleton instance will get started only once, so we might actually want to be notified with an exception.

Reply:
I could be convinced, but I'm leery of this because of the fact that this code is running in end user's apps; we'd basically be introducing the opportunity for something exactly like #15708 to happen here. If something in the future causes `start` to be called twice a small percentage of the time, and it slips out unnoticed, would prefer to just do the right thing to keep the app running rather than crashing. And here I think it makes sense that calling `start` a second time simply no-ops because the updates controller flow is already started.